tests/rpc_performance_test.ts

```diff
@@ -16,23 +16,30 @@ const test_config = {
         vn_batch_number: 2,
         vn_batch_size: 4096,
     },
-    timeout: 200000,
-    expected_sync_time: 600
+    timeout: 1000000,
+    sync_iterations: 50,
+    expected_average_sync_time: 650
 }
 
 async function single_rpc_performance(api:ApiPromise) {
-    const receiver_checkpoint = new Array<number>(manta_pay_config.shard_number);
-    receiver_checkpoint.fill(0);
-    const before_rpc = performance.now();
-    const data = await (api.rpc as any).mantaPay.pull_ledger_diff(
-        {receiver_index: new Array<number>(manta_pay_config.shard_number).fill(0), sender_index: 0},
-        BigInt(16384), BigInt(16384));
-    const after_rpc = performance.now();
```

Comment:
Not knowing the test framework, i assume the setup happens in that line

https://github.com/Manta-Network/Manta/pull/681/files/c8261bf4b29f8a1ef09386b96221c15a481d6ce5..43b1de6efd1c84c391b306359db3aa197c3fb0ed#diff-19d51960f59786f9c40532e38e3a902c64be0e21e3b84a46702bcb0718a20605R29

If that's not the case and setup is a one-time invocation that's constant, can't you just add that constant to the calculated timeout?